snowplow-demo-kotlin/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,49 @@
+<resources>
+    <string name="app_name">Tracker Demo</string>
+
+    <string name="main_demo_header">Android Tracker Demo</string>
```

Comment:
Nit: maybe mention "Kotlin" here in the title or distinguish them some other way in the UI? I got confused which demo app I was running when trying this out 😅